concurrency/suite.xml

```diff
@@ -1,29 +0,0 @@
-<!--
```

Comment:
Not sure about this, the suite.xml came from the tck runner starter/template from jakarta, and may be used to filter which tests are run. For instance it was me who temporarily excluded the signature tests to get them out of the way...

Do we have another way to do this already or in the works?

Reply:
TBH I was 50/50 on this. The TCK documentation said this which is why I removed it:

> Note: An implementation MUST run against all tests provided in the suite XML file unmodified for
an implementation to pass the TCK.

Reply:
What if a test challenge is approved, do they re-release the tck?

Reply:
To my knowledge, yes.

Reply:
Okay I added it back. I'm not sure how to read that remark of "Note: An implementation MUST run against all tests provided in the suite XML file unmodified for an implementation to pass the TCK.", but it's added back and it matches what is downloaded.